index.py

```diff
@@ -282,7 +282,7 @@
                     ],
                     options={
                         'class:':fink_classes, 'class=':fink_classes,
-                        'last:':['10', '100', '1000'], 'last=':['10', '100', '1000'],
+                        'last:':['10', '100', '1000'], 'last=':['1', '10', '100', '1000'],
```

Comment:
`'last:':['10', '100', '1000'],` should be `'last:':['1', '10', '100', '1000'],` to mirror options for `last=`